src/log-update.ts

```diff
@@ -10,37 +10,37 @@ export type LogUpdate = {
 };
 
 const createStandard = (
-	stream: Writable,
+	inkInstance: Ink,
 	{showCursor = false} = {},
 ): LogUpdate => {
 	let previousLineCount = 0;
-	let previousOutput = '';
 	let hasHiddenCursor = false;
 
+	const stream = inkInstance.options.stdout;
+
 	const render = (str: string) => {
 		if (!showCursor && !hasHiddenCursor) {
 			cliCursor.hide();
 			hasHiddenCursor = true;
 		}
 
 		const output = str + '\n';
-		if (output === previousOutput) {
+		if (output === inkInstance.lastOutput) {
 			return;
 		}
 
-		previousOutput = output;
+		inkInstance.lastOutput = output;
 		stream.write(ansiEscapes.eraseLines(previousLineCount) + output);
 		previousLineCount = output.split('\n').length;
 	};
 
 	render.clear = () => {
 		stream.write(ansiEscapes.eraseLines(previousLineCount));
-		previousOutput = '';
 		previousLineCount = 0;
```

Comment:
Won't clear inkInstance.lastOutput since it will trigger component rerender. Same for the incremental render part